ts/screens/authentication/SpidInformationRequestScreen.tsx

```diff
@@ -26,30 +30,23 @@ type ReduxMappedProps = {
 type OwnProps = {
   navigation: NavigationScreenProp<NavigationState>;
 };
-type Props = ReduxMappedProps & OwnProps;
+
+type Props = ReduxMappedProps & OwnProps & ContextualHelpInjectedProps;
 type State = {
-  isTosModalVisible: boolean;
+  isHelpVisible: boolean;
 };
 /**
  * A screen where the user can insert an email to receive information about SPID.
  */
 class SpidInformationRequestScreen extends React.Component<Props, State> {
   public state: State = {
-    isTosModalVisible: false
+    isHelpVisible: false
```

Comment:
why do you need this in the wrapped componen ?

Reply:
You're right, this isn't needed: I forgot to the rid of it (indeed, the entire state isn't needed)